book/forests.qmd

```diff
@@ -8,29 +8,45 @@ abstract: TODO (150-200 WORDS)
+
+The graphic highlights several core features of decision trees:
+
+1. They can model non-linear effects: The hierarchical structure allows for complex interactions between variables;
+2. They are highly interpretable: it is easy to visualise the tree and see how predictions are made;
+3. They perform variable selection: a very small subset of variables (2 out of 10) were used to train the model by carefully selecting the variables that would lead to the most heterogeneity in the resulting splits -- discussed next.
+
+To understand how random forests work, it is worth looking a bit more into the most important components of decision trees: splitting rules, stopping rules, and terminal node predictions.
 
 ![Demonstrating classification trees using the `mtcars`  [@datamtcars] dataset and the $\pkg{party}$  [@pkgparty] package. Ovals are leaves, which indicate the variable that is being split. Edges are branches, which indicate the cut-off at which the variable is split. Rectangles are terminal nodes and include information about the number of training observations in the node and the terminal node prediction.](Figures/forests/iris_tree.png){#fig-surv-ranfor fig-alt="TODO"}
 
-### Splitting Rules {.unnumbered .unlisted}
+#### Splitting and Stopping Rules {.unnumbered .unlisted}
+
+Precisely how the data partitions/splits are derived and which variables are utilised is determined by the *splitting rule*.
+The goal in each partition is to find two resulting leaves/nodes that have the greatest difference between them and thus the maximal homogeneity within each leaf, hence with each split, the data in each node should become increasingly similar.
+The splitting rule provides a way to measure the homogeneity within the resulting nodes.
+In regression, the most common splitting rule is to select a variable and cut-off (i.e., a threshold on the variable at which to separate observations) that minimises the mean squared error in the two potential resulting leaves.
 
-Precisely how the splits are derived and which variables are utilised is determined by the splitting rule.\footnote{Other methods for growing trees such as pruning are not discussed here as they are less relevant to random forests, which are primarily of interest. Instead see (e.g.) Breiman (1984)  [@Breiman1984].}
-In regression, the most common splitting rule is to select the cut-off for a given variable that minimises the mean squared error in each hypothetical resultant leaf.
-The goal is to find the variable and cutoff that leads to the greatest difference between the two resultant leaves and thus the maximal homogeneity within each leaf.
 For all decision tree and random forest algorithms going forward, let $L$ denote some leaf, then let $L_{xy}, L_x, L_y$ respectively be the set of observations, features, and outcomes in leaf $L$.
 Let $L_{y;i}$ be the $i$th outcome in $L_y$ and finally let $L_{\bar{y}} = \mean{L_{y;i}}$.
```

Comment:
the sum is from i to n here, but should be i in L_y, i.e. cardinality of the leaf